greedy_algorithms/boruvkas_minumum_spanning_tree.cpp

```diff
@@ -0,0 +1,208 @@
+﻿/**
+ * @author [Jason Nardoni](https://github.com/JNardoni)
+ * @file
+ *
+ * @brief
+ * Borůvkas Algorithm to find the Minimum Spanning Tree (https://en.wikipedia.org/wiki/Borůvka's_algorithm)
+ * 
+ * 
+ * @details
+ * Boruvka's algorithm is a greepy algorithm to find the MST by starting with small trees, and combining
+ * them to build bigger ones. It assumes that all nodes are connected.
+ *	1. Creates a group for every vertex.
+ *	2. looks through each edge of every vertex for the smallest weight. Keeps track
+ *		of the smallest edge for each of the current groups. 
+ *  3. Combine each group with the group it shares its smallest edge, adding the smallest
+ *		edge to the MST. 
+ *  4. Repeat step 2-3 until all vertices are combined into a single group.
+ * 
+*/
+
+#include <iostream>
+#include <vector>
+#include <cassert>
+#include <limits.h>
```

Comment:
```suggestion
#include <climits>
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not secure